frappe/core/doctype/communication_link/communication_link.py

```diff
@@ -27,3 +28,6 @@ class CommunicationLink(Document):
 
 def on_doctype_update():
 	frappe.db.add_index("Communication Link", ["link_doctype", "link_name"])
+	frappe.db.add_index(
+		"Communication Link", ["link_doctype", "link_name", "communication_date DESC", "parent"]
```

Comment:
@the-bokya you remember if `DESC` is required or not? IIRC you did something similar in lending 👀